SOOUM/SOOUM/Presentations/Intro/Launch/LaunchScreenViewReactor.swift

```diff
@@ -97,8 +106,23 @@ extension LaunchScreenViewReactor {
                 }
             }
     }
-}
     
+    private func fetchAppFlag() -> Observable<Mutation> {
+        let request: ConfigureRequest = .appFlag
+        return provider.networkManager.request(Bool.self, request: request)
+            .map { flag in
+                UserDefaults.standard.set(flag, forKey: "AppFlag")
+                print("@@@ AppFlag 성공", flag)
+                return Mutation.appFlag(flag)
+            }
+            .catch { _ in
+                print("@@@ AppFlag 안됨 true")
+                UserDefaults.standard.set(true, forKey: "AppFlag")
+                return .just(Mutation.appFlag(false))
```

Comment:
```suggestion
                UserDefaults.standard.set(flag, forKey: "AppFlag")
                Log.debug("AppFlag 성공", flag)
                return Mutation.appFlag(flag)
            }
            .catch { _ in
                Log.debug("AppFlag 안됨 true")
                UserDefaults.standard.set(true, forKey: "AppFlag")
                return .just(Mutation.appFlag(false))
```

디버깅을 위한 print 였다면 Log.debug로 변경해주시고, 단순 확인용이었다면 지우고 합쳐주세요.